goal_src/jak2/engine/sound/gsound.gc

```diff
@@ -550,9 +550,9 @@
   )
 
 (defun sound-volume-off ()
-  (set-setting! 'music-volume 'abs 0 0)
-  (set-setting! 'sfx-volume 'abs 0 0)
-  (set-setting! 'ambient-volume 'abs 0 0)
+  ;; (set-setting! 'music-volume 'abs 0 0)
```

Comment:
why comment these out?

Reply:
no idea why i did this

Reply:
ah, has to do with the confusion around how we have a `set-setting!` macro instead of just using the method, and how there's a build in form analysis set that looks for `set-setting`.

Reply:
the macro hides away the process register and global that are used 100% of the times.